cornac/models/hypar/recom_hypar.py

```diff
@@ -0,0 +1,771 @@
+import collections
+import os
+import pickle
+from collections import defaultdict
+from contextlib import nullcontext
+from copy import deepcopy
+
+from ..recommender import Recommender
+from ...data import Dataset
+
+
+class HypAR(Recommender):
+    def __init__(self,
```

Comment:
Please add doc strings for this model.

Reply:
You can refer to any model document, for example: https://github.com/PreferredAI/cornac/blob/e48153d4c01a816ede22b9f1dab7b30443d5ebac/cornac/models/lightgcn/recom_lightgcn.py#L24